screens/rag_screen.py

```diff
@@ -0,0 +1,27 @@
+import streamlit as st
+from utilz.pdf_extraction import list_available_pdfs, extract_text_from_pdf
+import google.generativeai as genai
+from utilz.llama_response import get_rag_response
+
```

Comment:
Ideally we should setup the rag for giving better context for story generation but that's not needed for a demo, so leaving it this way. If asked we will say that the point of this is so people can upload and query sleep related papers and data.